.devcontainer/devcontainer.json

```diff
@@ -1,20 +1,33 @@
-{
-    "name": "Ubuntu-21.04",
-    "dockerFile": "Dockerfile",
-    "settings": { 
-        "terminal.integrated.shell.linux": "/bin/bash"
-    },
-    "extensions": [
-		"ms-vscode.cpptools-themes",
-		"bbenoist.doxygen",
-		"streetsidesoftware.code-spell-checker",
-		"ms-vscode.cpptools",
-		"xaver.clang-format",
-		"twxs.cmake",
-		"ms-vscode.cmake-tools",
-		"eamodio.gitlens",
-		"davidschuldenfrei.gtest-adapter"
-	],
-    // Do not run as root. See https://aka.ms/vscode-remote/containers/non-root.
-    "remoteUser": "azure-sdk-for-cpp"
+﻿{
+  "name": "Ubuntu-20.04.VCPKG",
+  "dockerFile": "Dockerfile",
+  "settings": {
+    "terminal.integrated.shell.linux": "/bin/bash"
+  },
+  "extensions": [
+    "ms-vscode.cpptools-themes",
+    "bbenoist.doxygen",
+    "streetsidesoftware.code-spell-checker",
+    "ms-vscode.cpptools",
+    "xaver.clang-format",
+    "twxs.cmake",
+    "ms-vscode.cmake-tools",
+    "eamodio.gitlens",
+    "davidschuldenfrei.gtest-adapter"
+  ],
+  // Do not run as root. See https://aka.ms/vscode-remote/containers/non-root.
+  "remoteUser": "azure-sdk-for-cpp",
+  "containerEnv": {
+    "AZURE_CLIENT_ID": "",
+    "AZURE_CLIENT_SECRET": "",
+    "AZURE_KEYVAULT_HSM_URL": "",
+    "AZURE_KEYVAULT_URL": "",
+    "AZURE_LOG_LEVEL": "1",
+    "AZURE_TENANT_ID": "",
+    "AZURE_TEST_MODE": "PLAYBACK",
+    "CLIENT_OBJECTID": "",
+    // set or unset to run using vcpkg or not
+    "VCPKG_ROOT": "/vcpkg",
+    "VCPKG_INSTALLATION_ROOT": "/vcpkg"
```

Comment:
```suggestion
    "VCPKG_ROOT": "/vcpkg"
```

Reply:
We only need one.
Not sure if this would bring issue if vcpkg folder is not there